appveyor.yml

```diff
@@ -14,7 +14,7 @@ cache:
 
 branches:
   only:
-    - master
```

Comment:
I see AppVeyor hasn't passed this branch because this branch has been omitted. Maybe this PR should track both `master` and `development` for Appveyor and Travis until we've had a chance to complete the migration?